File: core/src/main/java/vn/loitp/restapi/flickr/model/photosetgetlist/Photosets.java

package vn.loitp.restapi.flickr.model.photosetgetlist;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Photosets {

    @SerializedName("page")
    @Expose
    private String page;
    @SerializedName("pages")
    @Expose
    private double pages;
    @SerializedName("perpage")
    @Expose
    private String perpage;
    @SerializedName("total")
    @Expose
    private double total;
    @SerializedName("photoset")
    @Expose
    private List<Photoset> photoset = null;

    public String getPage() {
        return page;
    }

    public void setPage(String page) {
        this.page = page;
    }

    public double getPages() {
        return pages;
    }

    public void setPages(double pages) {
        this.pages = pages;
    }

    public String getPerpage() {
        return perpage;
    }

    public void setPerpage(String perpage) {
        this.perpage = perpage;
    }

    public double getTotal() {
        return total;
    }

    public void setTotal(double total) {
        this.total = total;
    }

    public List<Photoset> getPhotoset() {
        return photoset;
    }

    public void setPhotoset(List<Photoset> photoset) {
        this.photoset = photoset;
    }

}

File: core/src/main/java/com/loitpcore/restApi/flickr/model/photoSetGetList/Photosets.kt
package com.loitpcore.restApi.flickr.model.photoSetGetList

import androidx.annotation.Keep
import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.loitp.core.base.BaseModel

/**
 * Created by Loitp on 04,August,2022
 * Galaxy One company,
 * Vietnam
 * +840766040293
 * freuss47@gmail.com
 */
@Keep
class Photosets : BaseModel() {
    @SerializedName("page")
    @Expose
    var page: String? = null

    @SerializedName("pages")
    @Expose
    var pages = 0.0

    @SerializedName("perpage")
    @Expose
    var perpage: String? = null

    @SerializedName("total")
    @Expose
    var total = 0.0

    @SerializedName("photoset")
    @Expose
    var photoset = ArrayList<Photoset>()
}
